packages/api/bin/serve.js

```diff
@@ -276,6 +307,54 @@ async function serveDistributionApi(stackName = defaultLocalStackName, done) {
   return distributionApp.listen(port, done);
 }
 
+
+/**
+ * erase all dynamoDB tables
+ * @param {string} stackName - defaults to local stack, localrun
+ * @param {string} systemBucket - defaults to 'localbucket', localrun
+ */
+async function eraseDynamoTables(stackName, systemBucket) {
+  setTableEnvVariables(stackName);
+  process.env.system_bucket = systemBucket;
+  process.env.stackName = stackName;
+
+  // Remove all data from tables
+  const providerModel = new models.Provider();
+  const collectionModel = new models.Collection();
+  const rulesModel = new models.Rule();
+  const executionModel = new models.Execution();
+  const granulesModel = new models.Granule();
+  const pdrsModel = new models.Pdr();
+
+  try {
+    await rulesModel.deleteRules();
+    await Promise.all([
+      collectionModel.deleteCollections(),
+      providerModel.deleteProviders(),
+      executionModel.deleteExecutions(),
+      granulesModel.deleteGranules(),
+      pdrsModel.deletePdrs()
+    ]);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+
+/**
+ * Erases DynamoDB tables and resets Elasticsearch
+ *
+ * @param {string} stackName - defaults to local stack, localrun
+ * @param {string} systemBucket - defaults to 'localbucket', localrun
```

Comment:
This value never defaults to `localrun`, correct?

Reply:
Yup

Reply:
There were a couple of others too.